configuration/configuration.go

```diff
@@ -36,6 +38,21 @@ type DB struct {
 type Replicator struct {
 	Addr            string `insconfig:"127.0.0.1:5678| The gRPC server address"`
 	MaxTransportMsg int    `insconfig:"1073741824| Maximum message size the client can send"`
+	Auth            Auth
+}
+
+// Auth represents the authentication of the Platform
+type Auth struct {
```

Comment:
just imho, this whole struct must be optional, require field looks strange, like this is not required but you must fill it with any garbage